hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieAppendHandle.java

```diff
@@ -508,10 +509,7 @@ protected void doWrite(HoodieRecord record, Schema schema, TypedProperties props
       flushToDiskIfRequired(record, false);
       writeToBuffer(record);
     } catch (Throwable t) {
-      // Not throwing exception from here, since we don't want to fail the entire job
-      // for a single record
-      writeStatus.markFailure(record, t, recordMetadata);
-      LOG.error("Error writing record " + record, t);
+      throw new HoodieInsertException("Error writing record " + record, t);
```

Comment:
In Flink SQL, option `write.ignore.failed` controls the throwing behavior of errors.

Reply:
revert changed in HoodieAppendHandle @danny0405

Reply:
Does Spark have similiar option?

Reply:
spark seem not the config